packages/rest/test/acceptance/routing/routing.acceptance.ts

```diff
@@ -119,6 +119,47 @@ describe('Routing', () => {
     );
   });
 
+  it('allows controllers to use method DI with mixed params', async () => {
+    class MyController {
+      @get('/greet')
+      greet(
+        @param.query.string('name') name: string,
+        @inject('hello.prefix') prefix: string,
+      ) {
+        return `${prefix} ${name}`;
+      }
+    }
+    const app = givenAnApplication();
+    app.bind('hello.prefix').to('Hello');
+    const server = await givenAServer(app);
+    givenControllerInApp(app, MyController);
+    return (
+      whenIMakeRequestTo(server)
+        .get('/greet?name=world')
+        // Then I get the result `hello world` from the `Method`
```

Comment:
I think this comment can be removed, since IMO the test is clear about what we get back.

Reply:
Sure. That was copied/pasted.